run_backpack.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+###
+# Default settings
+###
+
+CLEANUP=true
+PRIVILEGED=false
+ACCOUNT=default
+NAMESPACE=backpack
+UUID=`uuidgen`
+LABEL_NAME=""
+LABEL_VALUE=""
+
+while getopts s:p:n:xa:t:c:u:l:v:h flag
+do
+    case "${flag}" in
+        s) ES_SERVER=${OPTARG};;
+        p) ES_PORT=${OPTARG};;
+        n) NAMESPACE=${OPTARG};;
+        x) PRIVILEGED=true;;
```

Comment:
It would be great to have an IMAGE argument to customize the backpack image

Reply:
I like that idea. updated